core/modules/taxonomy/views/taxonomy.views.inc

```diff
@@ -185,6 +184,22 @@ function taxonomy_views_data() {
     ),
   );
 
+  $data['taxonomy_term_data']['term_tid_depth'] = array(
+    'help' => t('Display term ionly f it has the requested depth.'),
+    'real field' => 'tid',
+    /*
+    'argument' => array(
+      'title' => t('Taxonomy term depth'),
+      'handler' => 'views_handler_argument_term_depth',
+      'accept depth modifier' => TRUE,
+    ),
+    */
```

Comment:
This is the argument. I started with the filter. If we can get that working we'll need this later ;)